src/CaptivePortal.cpp

```diff
@@ -137,13 +262,13 @@ void handleWebSocketClientMessage(std::uint8_t socketId, WStype_t type, std::uin
   }
 
   String typeStr = doc["type"];
-  if (typeStr.length() == 0) {
+  if (typeStr.isEmpty()) {
     ESP_LOGE(TAG, "Message type is missing");
     return;
   }
 
-  if (typeStr == "startScan") {
-    WiFiManager::StartScan();
+  if (typeStr == "wifi") {
+    handleWebSocketClientWiFiMessage(doc);
   } /* else if (typeStr == "connect") {
```

Comment:
Commented out code? L272-284

Reply:
Removed alot of it, some left in for future implementation.